Inseok_45/controllers/userController.js

```diff
@@ -20,4 +25,18 @@ const signUp = async ( req, res ) => {
     };
 };
 
-module.exports = { signUp };
\ No newline at end of file
+const getUserById = async (req, res) => {
```

Comment:
```suggestion
const getUserById = async (req, res) => {
```
함수 이름 signIn 혹은 login으로 변경해주세요~